frontend/amundsen_application/static/js/components/ResourceListItem/styles.scss

```diff
@@ -14,6 +14,17 @@ $resource-item-height: 110px;
   padding: $spacer-3;
   text-decoration: none;
 
+  &.is-disabled {
+    pointer-events: none;
+    cursor: not-allowed;
+    .resource-icon {
+      filter: grayscale(80%);
```

Comment:
Hmmm... Can you tell me more about why we use this? It is a non-conventional way.

Reply:
I needed to add some sort of visual indicator that the table item is not actionable (won't go to a table page), and it felt like just the table name being gray was not enough. There is a ss showing what it looks like. Is there a better way of doing this?